src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -672,10 +696,10 @@ class MessageHelper {
       ApplicationManager& app_mngr);
 
   /**
-  * @brief Stores whether each choice in a set has the vrCommands parameter
-  * MIXED means some choices have vrCommands and others don't
-  * ALL means all do, NONE means none do
-  */
+   * @brief Stores whether each choice in a set has the vrCommands parameter
+   * MIXED means some choices have vrCommands and others don't
```

Comment:
@mked-luxoft  is it auto formating of clang-style check? If not ( i think not) please remove.